spec/requests/webhook_spec.rb

```diff
@@ -0,0 +1,77 @@
+require 'rails_helper'
+
+RSpec.describe 'Pingometer Webhook', type: :request do
+  it 'is persisted appropriately in the database' do
```

Comment:
@Mr0grog Could you please review this rather long spec and let me know if the logic outlined is correct in regards to how events should be stored, coalesced into incidents or discarded.